python/setup.py

```diff
@@ -22,9 +22,9 @@ def finalize_options(self):
 
 _kastore_module = Extension(
     "_kastore",
-    sources=["_kastoremodule.c", "lib/kastore.c"],
+    sources=["_kastoremodule.c", "../c/kastore.c"],
```

Comment:
That won't work in general, this is done in the context of the distribution tarball.

Reply:
Reverted to use the git symlink method